internal/docs/generated/fndocs/docs.go

```diff
@@ -40,6 +26,28 @@ var FnExamples = `
   kpt fn run DIR/
 `
 
+var EvalShort = `Locally execute one or more functions in containers`
+var EvalLong = `
+  kpt fn eval DIR [flags]
+
+If the container exits with non-zero status code, eval will fail and print the
+container ` + "`" + `STDERR` + "`" + `.
+
+  DIR:
+    Path to a package directory.  Defaults to stdin if unspecified.
+`
+var EvalExamples = `
+  # read the Resources from DIR, provide them to a container my-fun as input,
+  # write my-fn output back to DIR
+  kpt fn eval DIR/ --image gcr.io/example.com/my-fn
+
+  # provide the my-fn with an input ConfigMap containing ` + "`" + `data: {foo: bar}` + "`" + `
+  kpt fn eval DIR/ --image gcr.io/example.com/my-fn:v1.0.0 -- foo=bar
+
+  # run the function in FUNCTIONS_CONFIG_PATH against the Resources in DIR
```

Comment:
This is not accurate.

```
run the function with a function config file
```

Also double check if this arg in CAPS is consistent.

Reply:
Updated.

This arg was in CAPS before.